src/writeInterface.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Generates Solidity interface from ABI and writes it to a specified stream
+ * @param {[]} functions ABI descriptions of function signatures
+ * @param {string} pragma Solidity pragma statement
+ * @param {WritableStream} stream stream to write to
+ */
+function writeInterface(functions, pragma, stream = process.stdout) {
+  stream.write(`${'// SPDX-License-Identifier: GPL-3.0'.gray}
```

Comment:
- syntax coloured output might mess with compatibility - e.g. if you run this in a non-POSIX compliant terminal it may render strange characters.
- additionally, when the intent is to write to a file, this effort is likely to be discarded.

therefore, it would be better to "gate" this using a CLI flag.